tools/pipelines/templates/include-conditionally-run-stress-tests.yml

```diff
@@ -80,4 +84,4 @@ stages:
             testFileTarName: ${{ package.testFileTarName }}
             testCommand: ${{ package.testCommand }}
             env:
-              FUZZ_STRESS_RUN: true
+              FUZZ_STRESS_RUN: ${{ parameters.stressMode }}
```

Comment:
i think the values in package.json scripts passed via cross-env will override this one. you don't have to fix that now, but i'd at least add a comment that the override order here isn't really right.